### ISSUE TITLE
Docs: Warping tracks

### ISSUE THREAD

**Issue by Lestropie** [MRtrix3/mrtrix3#780]
I'm sure this has come up a few times on the forum, and it's in the 0.2 docs, but hasn't been added to the *MRtrix3* docs yet. Instructions for warping streamlines to template space. Probably base off `mrregister`-derived warps but include instructions for other packages where possible.

**Comment by jdtournier** [comment]
Yes, I'll pile that onto my todo list... Unless anyone else feels like taking it on...?
